rails_db.gemspec

```diff
@@ -15,24 +15,38 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,lib,test}/**/*", "rails_db.gemspec", "Gemfile", "Gemfile.lock", "MIT-LICENSE", "Rakefile", "README.rdoc", "bin/rails_db", "bin/railsdb", "bin/runsql"]
   s.test_files = Dir["test/**/*"]
 
-  s.executables   = ["railsdb", "rails_db", 'runsql']
+  s.executables   = ['railsdb', 'rails_db', 'runsql']
```

Comment:
did you tried to run rails_db ? (I mean executables if works too?)

Reply:
and runsql

Reply:
agh! You're right, the executables don't work. Only the GUI. Sorry i didn't catch that before.

standalone.rb is throwing an error when i try to run `railsdb`, it says that 
```
/home/vagrant/.rvm/gems/ruby-3.0.4/gems/rack-2.2.4/lib/rack/handler/webrick.rb:26:in `run': 
wrong number of arguments (given 2, expected 1) (ArgumentError)
```
If I eliminate the options hash in line 46 of standalone.rb, it seems to launch but is not available at port `:12345`, it uses the WEBrick default port `:8080`. However, the `Rack::Handler::WEBrick` method `run` clearly should accept two arguments, `app` and `options`.
```ruby
module Rack
  module Handler
    class WEBrick < ::WEBrick::HTTPServlet::AbstractServlet
      def self.run(app, **options)
        environment  = ENV['RACK_ENV'] || 'development'
        default_host = environment == 'development' ? 'localhost' : nil

        if !options[:BindAddress] || options[:Host]
          options[:BindAddress] = options.delete(:Host) || default_host
        end
        options[:Port] ||= 8080
        if options[:SSLEnable]
          require 'webrick/https'
        end

        @server = ::WEBrick::HTTPServer.new(options)
        @server.mount "/", Rack::Handler::WEBrick, app
        yield @server  if block_given?
        @server.start
      end

      def self.valid_options
        environment  = ENV['RACK_ENV'] || 'development'
        default_host = environment == 'development' ? 'localhost' : '0.0.0.0'

        {
          "Host=HOST" => "Hostname to listen on (default: #{default_host})",
          "Port=PORT" => "Port to listen on (default: 8080)",
        }
      end
...
```
If you have a chance, maybe you can help me debug why these two executables aren't launching.